### PR TITLE
[SDPA-4970] enable eslint rules and fix issues

### PR DESCRIPTION
**Pull request by rubyllamadora**
## Motivation and Context

**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-4970

## Changed

1.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist

- [ ] I've added relevant changes to the documentation.
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [x] My change requires a template update for create-ripple-app.
- [x] I have added template update script for next release.

### CODE REVIEW

File: packages/create-ripple-app/generators/updates/1.33.0/saofile.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+// allow debugger during development
+'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+// disable converting single item array to use dot-notation
+'dot-notation': 'off',
+// allow ternary operator in single line
+'multiline-ternary': ['error', 'never'],
+// currently being disabled as it's complaining on for loop
+'prefer-const': 'off',
+// we should use single quote for consistency, also allowing backticks
+quotes: ['off', 'single', { allowTemplateLiterals: true }]`
```
Comment (rubyllamadora) [inline packages/create-ripple-app/generators/updates/1.33.0/saofile.js:30]: I've tested this on vic-gov-au. It requires fixing other rule e.g. `vue/no-unused-components`, `vue/no-unused-components`. I'll create a PR in there basing off develop.

**Comment by rubyllamadora** [comment]
This is probably hard to review having a lot of changed files. I'll break this down into one rule enabled at a time. 
cc @dylankelly

**Comment by dylankelly** [comment]
@rubyllamadora - I'm not ready to implement this and I want to significantly change the way linting is implemented - I think this will need to be parked for now i'm afraid

**Comment by dylankelly** [comment]
Changing Eslint rules now is going to be a massive headache - I'm going to postpone this for a while - I dont think this work will be necessary after that.

**Pull request closed without merging**